tlklan/index.php

```diff
@@ -10,9 +10,10 @@
 }
 
 // Enable error reporting and display errors eventually
-error_reporting(E_ALL);
+error_reporting(E_ERROR);
```

Comment:
Why change this? This means eventual problems won't even be logged. `display_errors` controls whether errors are shown to the user or not, regardless of the `error_reporting` level.

Reply:
Because phpseclib uses undefined constants and php 7.2 likes to whine about it to the user. maybe when phpseclib stops using deprecated features we can change it back.